src/cmd/tools/m3ctl/client/checker.go

```diff
@@ -0,0 +1,21 @@
+package client
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"go.uber.org/zap"
+)
+
+func checkForAndHandleError(url string, resp *http.Response, zl *zap.Logger) error {
+	zl.Info(fmt.Sprintf("resp.StatusCode:%d:\n", resp.StatusCode))
+	if resp.StatusCode > 299 {
+		dat, _ := ioutil.ReadAll(resp.Body)
+		if dat != nil {
+			zl.Info(string(dat))
+		}
+		return fmt.Errorf("error from m3db:%s:url:%s:", resp.Status, url)
```

Comment:
Maybe space this out a bit? Usually we use the form "some message: foo=bar, baz=qaz" 
```
fmt.Errorf("error response: status=%s, url=%s", resp.Status, url)
```